Automattic/wp-calypso
# Hosting Overview page

Related to https://github.com/Automattic/dotcom-forge/issues/6384

## Proposed Changes

* Implement the hosting overview section described on the linked issue.
* The nav, tabs, and header for the whole page will be added later (see https://github.com/Automattic/dotcom-forge/issues/6384#issuecomment-2035181357).
* Related to the design, there are the following changes:
  * On the plan card, I've replaced `$25/mo` with `$25 per month` in order to be able to reuse the `PlanPrice` component.
  * On the plan card, the `Add 100/150GB` links have been removed. See https://github.com/Automattic/dotcom-forge/issues/6384#issuecomment-2050203356.
  * On the quick actions card, the `Explore Marketplace` link has been removed. See https://github.com/Automattic/dotcom-forge/issues/6384#issuecomment-2037249402

## Testing Instructions

* Open the live preview
* Go to `/hosting-overview`
* Check that it looks like this:

![image](https://github.com/Automattic/wp-calypso/assets/8511199/4da94512-bcdb-4deb-81b1-44e2e85630eb)

## Pre-merge Checklist
